package.json

```diff
@@ -8,14 +8,14 @@
   },
   "scripts": {
     "start": "yarn build:dev dev --apply-lavamoat=false --snow=false",
-    "start:mv3": "ENABLE_MV3=true yarn build:dev dev --apply-lavamoat=false",
+    "start:mv3": "ENABLE_MV3=true yarn build:dev dev --apply-lavamoat=false --snow=false",
     "start:flask": "yarn start --build-type flask",
     "start:mmi": "yarn start --build-type mmi",
     "start:lavamoat": "yarn build:dev dev --apply-lavamoat=true",
     "dist": "yarn build dist",
     "dist:mv3": "ENABLE_MV3=true yarn build dist",
     "dist:mmi": "yarn dist --build-type mmi",
-    "dist:mmi:debug": "yarn dist --build-type mmi --apply-lavamoat=false",
+    "dist:mmi:debug": "yarn dist --build-type mmi --apply-lavamoat=false --snow=false",
```

Comment:
Dear reviewer: does the addition of `--snow=false` seem appropriate here?

Reply:
@zone-live